### DISCUSSION TITLE
Self Hosting - Login page doesn't "work"

### DISCUSSION THREAD

**Discussion by SteveChurch** [supabase/supabase#12035] [Questions]
So, I now have Supabase running on Digital Ocean seamlessly, fully managed infrastructure in code. 

However, when I go to my Supabase Studio it flashes up the login page and then just takes me straight in.. Not ideal and not that secure. I could set up some firewall rules for my IP address but I'd prefer to have the auth working. 

Has anyone else experienced this with self-hosting?

**Comment by GaryAustin1** [comment]
https://supabase.com/docs/guides/self-hosting/docker#securing-the-dashboard
`....doesn't include a management database for managing users and logins. If you plan to deploy the Studio to the web we suggest you put it behind a web proxy with Basic Auth or hide it behind a VPN.`

**Reply by SteveChurch** [reply]
Oh that's a shame. Why is that? I assume to stop people cloning the core business model with the management features, but a shame as it would be nice to have something more than Basic Auth. 

I'll move to securing it behind a VPN connection. 

Thank you.

**Comment by miguelgargallo** [comment]
it seems to be in complete while all database open-source have this feature, which is basic
